src/derivatives/introduction.md

```diff
@@ -4,9 +4,28 @@ Derivatives are outputs of common processing pipelines, capturing data and
 meta-data sufficient for a researcher to understand and (critically) reuse those
 outputs in subsequent processing.
 Standardizing derivatives is motivated by use cases where formalized
-machine-readable access to processed data enables higher level processing.
+machine-readable access to processed data enables higher-level processing.
+
+The following sections cover additions to and divergences from "raw" BIDS. Raw data are
+data that have been curated to BIDS, i.e. a BIDS “raw” dataset does not have another BIDS
+dataset as a source. If you are in doubt if your dataset is "raw" or
+is "derivatives" ask yourself the following question: "Are my data derived from at least
+one other valid BIDS dataset?" if the answer is yes, then this is a BIDS derivative dataset.
+
+Examples:
+
+A defaced T1w image would typically be made during the curation process and is thus under raw
+
+`sourcedata/private/sub-01/anat/sub-01_T1w.nii.gz`
+`sub-01/anat/sub-01_desc-defaced_T1w.nii.gz`
```

Comment:
Using triple-ticks to make these one code block. Also, `desc-` is not permitted in raw:

````suggestion
```
sourcedata/private/sub-01/anat/sub-01_T1w.nii.gz
sub-01/anat/sub-01_T1w.nii.gz
```
````